modules/lvms-unsupported-devices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-unsupported-devices_{context}"]
+= Devices not supported by {lvms}
+
+When you are adding the paths to the devices in the `deviceSelector` section of the `LVMCluster` YAML, ensure that the devices are supported by {lvms}. If you add paths to the unsupported devices, then {lvms} excludes the devices to avoid complexity in managing logical volumes.
+
+If you do not specify any path to the device, then {lvms} adds only the unused devices that it supports. 
```

Comment:
```suggestion
If you do not specify any path in the `deviceSelector`, then {lvms} adds only the unused devices that it supports. 
```

Reply:
@suleymanakbas91 
This change has been incorporated. PTAL.
Thanks!